cli/azd/cmd/sets.go

```diff
@@ -96,31 +111,36 @@ var CommonSet = wire.NewSet(
 
 var InitCmdSet = wire.NewSet(
 	CommonSet,
+	newCredential,
 	newAzCliFromOptions,
```

Comment:
Recommend creating a new set for `AzCliSet` that includes credential + azcli

Reply:
Done.